kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue

```diff
@@ -85,12 +85,15 @@
   import responsiveElementMixin from 'kolibri.coreVue.mixins.responsiveElementMixin';
   import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
   import CoreFullscreen from 'kolibri.coreVue.components.CoreFullscreen';
-  import urls from 'kolibri.urls';
   import PdfPage from './PdfPage';
   // Source from which PDFJS loads its service worker, this is based on the __publicPath
   // global that is defined in the Kolibri webpack pipeline, and the additional entry in the PDF
   // renderer's own webpack config
-  PDFJSLib.PDFJS.workerSrc = urls.static(`${__kolibriModuleName}/pdfJSWorker-${__version}.js`);
+
+  /* eslint-disable no-undef */
+  PDFJSLib.GlobalWorkerOptions.workerSrc = __webpack_public_path__ + `pdfJSWorker-${__version}.js`;
```

Comment:
Seems we need to add this to globals, both for linting and for jest.

Can add a line here: https://github.com/learningequality/kolibri/blob/develop/packages/kolibri-tools/.eslintrc.js#L51 for `__webpack_publis_path__` to make eslint know this is an allowed global.

And here can do the same for jest globals: https://github.com/learningequality/kolibri/blob/develop/packages/kolibri-tools/jest.conf/index.js#L21 just set the value to a string value `'webpack_public_path'`.

Reply:
This was my fault as I mis-directed @AlexVelezLl after seeing that just skipping linting for this was used elsewhere! Thanks for the _correct_ way to deal with it :)